server/api/rounds.py

```diff
@@ -256,13 +256,13 @@ def cvrs_for_contest(contest: Contest) -> supersimple.CVRS:
 
 def sampled_ballot_interpretations_to_cvrs(contest: Contest) -> supersimple.CVRS:
     def interpretation_to_cvr_value(interpretation: Interpretation) -> int:
-        if interpretation == Interpretation.BLANK:
-            return 0
-        elif interpretation == Interpretation.VOTE:
+        if interpretation == Interpretation.VOTE:
             return 1
+        elif interpretation == Interpretation.BLANK:
+            return 0  # TODO make this a vote for the loser?
```

Comment:
No. this is right. We're saying that this wasn't a vote, therefore the most it can contribute is a one-vote misstatement. If we count it as a vote for the loser, then it potentially becomes a two-vote misstatement....